fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -48,6 +48,29 @@
 #include "flow/TDMetric.actor.h"
 #include "flow/actorcompiler.h"  // This must be the last #include.
 
+ACTOR Future<Void> broadcastTxnRequest(TxnStateRequest req, int sendAmount, bool sendReply) {
```

Comment:
Is the reason this doesn't return a list of failures that any failures would cause a re-recruitment anyway?

Reply:
yes, the master is separately checking for proxy errors, so this message uses reliable delivery